### PR TITLE
Create ui-c8y-1019-20-4-graft-release-cd-fixed-new-line-characters-for-results-in-shell.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by dafk

Ticket: [DM-3505](https://cumulocity.atlassian.net/browse/DM-3505)
Original PR: [6112](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6112)

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1019-20-4-graft-release-cd-fixed-new-line-characters-for-results-in-shell.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: [GRAFT][release/cd] Fixed new line characters for results in shell operation.  (#6112)
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1019-20-4-graft-release-cd-fixed-new-line-characters-for-results-in-shell.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Fixed new line characters for results in shell operation
```

File: content/change-logs/device-management/ui-c8y-1019-20-4-graft-release-cd-fixed-new-line-characters-for-results-in-shell.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3505
+version: 1019.20.4
+---
+[GRAFT][release/cd] Fixed new line characters for results in shell operation.  (#6112)
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1019-20-4-graft-release-cd-fixed-new-line-characters-for-results-in-shell.md:17]: The Claude AI suggested to use the following body:
```suggestion
In the Cumulocity IoT shell, commands executed through the shell operation previously returned results with missing new line characters, making the output difficult to read. This issue has now been resolved. The shell operation now correctly includes new line characters in the command output as expected. This improvement makes the results of shell commands much easier to read and interpret for users working with the shell operation.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/device-management/ui-c8y-1019-20-4-graft-release-cd-fixed-new-line-characters-for-results-in-shell.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fixed new line characters for results in shell operation
```
Comment (dafkk) [inline content/change-logs/device-management/ui-c8y-1019-20-4-graft-release-cd-fixed-new-line-characters-for-results-in-shell.md:3]:
```suggestion
title: Fixed missing newline characters in the result section in the shell tab on the device details page.
```

File: content/change-logs/device-management/ui-c8y-1019-20-4-graft-release-cd-fixed-new-line-characters-for-results-in-shell.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3505
+version: 1019.20.4
+---
+In the Cumulocity IoT shell, commands executed through the shell operation previously returned results with missing new line characters, making the output difficult to read. This issue has now been resolved. The shell operation now correctly includes new line characters in the command output as expected. This improvement makes the results of shell commands much easier to read and interpret for users working with the shell operation.
```
Comment (dafkk) [inline content/change-logs/device-management/ui-c8y-1019-20-4-graft-release-cd-fixed-new-line-characters-for-results-in-shell.md:17]:
```suggestion
In the Cumulocity IoT device details shell tab, commands executed through the shell operation previously displayed result template with missing new line characters, making it difficult to read. This issue has now been resolved. The shell operation now correctly includes newline characters in the command output as expected. This improvement makes the results of shell commands much easier to read and interpret for users working with the shell operation.
```

**Review by dafkk — APPROVED** [review]

File: content/change-logs/device-management/ui-c8y-1019-20-4-graft-release-cd-fixed-new-line-characters-for-results-in-shell.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fixed missing newline characters in the result section in the shell tab on the device details page.
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1019-20-4-graft-release-cd-fixed-new-line-characters-for-results-in-shell.md:3]:
```suggestion
title: Command output of shell operations in the Shell tab now correctly includes newline characters
```

**Review by l2p- — APPROVED** [review]

**Pull request merged**